### PR TITLE
Add Plaid Credentials API

### PR DESCRIPTION
**Pull request by slavcodev**

### CODE REVIEW

File: openapi/components/schemas/Credentials/PlaidCredentials/PlaidCredential.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    description: Plaid Credential
+    required:
+      - clientId
+      - secretToken
+    properties:
+      clientId:
+        type: string
+        description: The Plaid client ID
+        example: 5efc712byyf502zz11f9bexx
+      secretToken:
```
Comment (VladimirBerdnik) [inline openapi/components/schemas/Credentials/PlaidCredentials/PlaidCredential.yaml:13]: Does it have read and write access? Shouldn't the format be `password`?

**Review by ashkarpetin — DISMISSED** [review]

**Review by VladimirBerdnik — DISMISSED** [review]

**Comment by slavcodev** [comment]
php please review

**Review by ashkarpetin — DISMISSED** [review]

File: openapi/users.yaml [hunk 2]
```diff
@@ -244,6 +244,7 @@ x-tagGroups:
       - Tracking
       - Websites
       - Integrations
+      - Plaid Credentials
```
Comment (adamaltman) [inline openapi/users.yaml:247]:
```suggestion
      - Plaid credentials
```

File: openapi/paths/credential-hashes@plaid@{hash}.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            $ref: ../components/schemas/Credentials/PlaidCredentials/PlaidCredential.yaml
+    '401':
+      $ref: ../components/responses/Unauthorized.yaml
+    '403':
+      $ref: ../components/responses/Forbidden.yaml
+    '404':
+      $ref: ../components/responses/NotFound.yaml
+patch:
+  tags:
+    - Plaid Credentials
```
Comment (adamaltman) [inline openapi/paths/credential-hashes@plaid@{hash}.yaml:33]:
```suggestion
    - Plaid credentials
```

File: openapi/paths/credential-hashes@plaid@{hash}.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
+parameters:
+  - $ref: ../components/parameters/organizationId.yaml
+  - $ref: ../components/parameters/hash.yaml
+get:
+  tags:
+    - Plaid Credentials
```
Comment (adamaltman) [inline openapi/paths/credential-hashes@plaid@{hash}.yaml:6]:
```suggestion
    - Plaid credentials
```

File: openapi/paths/credential-hashes@plaid.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
+parameters:
+  - $ref: ../components/parameters/organizationId.yaml
+post:
+  tags:
+    - Plaid Credentials
```
Comment (adamaltman) [inline openapi/paths/credential-hashes@plaid.yaml:5]:
```suggestion
    - Plaid credentials
```

File: openapi/components/schemas/Credentials/PlaidCredentials/PlaidCredential.yaml [hunk 6]
```diff
@@ -0,0 +1,37 @@
+allOf:
+  - $ref: ../Credential.yaml
+  - type: object
+    description: Plaid Credential
```
Comment (adamaltman) [inline openapi/components/schemas/Credentials/PlaidCredentials/PlaidCredential.yaml:4]:
```suggestion
    description: Plaid credential
```

**Review by ashkarpetin — DISMISSED** [review]

File: openapi/components/schemas/Credentials/PlaidCredentials/PlaidCredential.yaml [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+      secretToken:
+        type: string
+        description: The Plaid secret token
+        format: password
+        example: d184cfdgdb4597dfsa9365f59356b
+      useStripe:
+        type: boolean
+        description: |
+          Whether the Stripe integration is enabled for Plaid account.
+          This setting will enable creation Stripe bank account tokens.
```
Comment (adamaltman) [inline openapi/components/schemas/Credentials/PlaidCredentials/PlaidCredential.yaml:28]:
```suggestion
          This setting will enable creating Stripe bank account tokens.
```

**Review by adamaltman — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**